app/src/main/AndroidManifest.xml

```diff
@@ -3,20 +3,22 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.ecosystem.kin.app">
 
-    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.INTERNET"/>
 
     <application
         android:allowBackup="false"
-        tools:replace="android:allowBackup"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
+        android:name=".App"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
-        android:name=".App"
-        tools:ignore="GoogleAppIndexingWarning">
+        tools:ignore="GoogleAppIndexingWarning"
+        tools:replace="android:allowBackup">
 
         <!--"test environment is only for internal usage, you should use only beta or production as you can see below"-->
-        <meta-data android:name="com.kin.ecosystem.sdk.EnvironmentName" android:value="test"/>
+        <meta-data
+            android:name="com.kin.ecosystem.sdk.EnvironmentName"
+            android:value="test"/>
```

Comment:
should be beta - developer should use test - In any case I'm going to fix this as part of productFlavor task